### PR TITLE
Release v4.4.2

### PR DESCRIPTION
**Pull request by Michiel-s**

### CODE REVIEW

**Comment by Michiel-s** [comment]
@hanjoosten, I've cleaned up and improved our CI scripts. Most important one is fixing/re-enabling the build push to Docker Hub. Last 5 releases were not pushed to Docker Hub.

To further improve the speed to release, I think we should let go of the development branch.

The release flow need to change a bit then.
* Current flow: PR to master creates a commit on master which triggers CI flow to create a Github release (fully automated)
* Target flow: PR to master creates a commit on master which triggers our regular CI checks and builds. Including build push of `latest` to Docker Hub. A manually created Github release, which triggers CI to attach artefacts AND build pushes to Docker Hub using semver of tag.

Basically we have the master branch as a stable latest version of our code. Using tags we identity releases.

But that is for some other time and to discuss further
